i18n/en.json

```diff
@@ -536,6 +536,38 @@
     "translation": "echo"
   },
   {
+    "id": "api.command_channel_header.desc",
+    "translation": "Edit the channel header."
+  },
+  {
+    "id": "api.command_channel_header.hint",
+    "translation": "[text]"
+  },
+  {
+    "id": "api.command_channel_header.name",
+    "translation": "header"
+  },
+  {
+    "id": "api.command_channel_header.message.app_error",
+    "translation": "A message must be provided with the /header command."
```

Comment:
The command in the server side wait for a message. I left here because if others implement their own client they might use that. however, if you think this is not necessary I can remove.